torch/testing/_internal/autocast_test_lists.py

```diff
@@ -318,6 +318,13 @@ def __init__(self, dev):
             ("conv_transpose2d", conv_args_fp32[1]),
             ("conv_transpose3d", conv_args_fp32[2]),
             ("prelu", pointwise0_fp32 + element0_fp32),
+            ("_native_multi_head_attention", (torch.randn((n, n, n), device=dev, dtype=torch.bfloat16),
```

Comment:
Shouldn't you be passing in fp32 arguments, ormaybe I am misunderstanding how this test works

Reply:
Oh, you are right! Thanks and modified.